src/sass/partials/layouts/_content-card.scss

```diff
@@ -0,0 +1,35 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  margin: 2rem 0 0;
+  width: 100%;
+
+
+  @include breakpoint(setting-get('two grid cards')) {
+    margin-right: 4%;
```

Comment:
Where does this 4% come from? That's not in line with _any_ of our rhythm anywhere. We've pretty consistently been doing spacing using `rem` and fixed sizes. Why the sudden change? And why this percentage?